src/app/pages/splash/containers/splash-page.component.ts

```diff
@@ -91,15 +91,12 @@ export class SplashPageComponent {
                   !lastUpdate
                 ) {
                   console.log('[SPLASH] Scheduling Notifications.')
-                  return this.notificationService.setNextXNotifications(
-                    DefaultNumberOfNotificationsToSchedule
-                  )
+                  return this.configService.rescheduleNotifications()
```

Comment:
Why reschedule ?? We just need to schedule them again. The server will take care of everything else. If we cancel and schedule again then unnecessary overhead on the server (will need to first remove from DB all the notifs and then add them again).